django/contrib/gis/gdal/prototypes/geom.py

```diff
@@ -112,7 +122,7 @@ def topology_func(f):
 get_area = double_output(lgdal.OGR_G_GetArea, [c_void_p])
 get_centroid = void_output(lgdal.OGR_G_Centroid, [c_void_p, c_void_p])
 get_dims = int_output(lgdal.OGR_G_GetDimension, [c_void_p])
-get_coord_dim = int_output(lgdal.OGR_G_GetCoordinateDimension, [c_void_p])
+get_coord_dim = int_output(lgdal.OGR_G_CoordinateDimension, [c_void_p])
```

Comment:
Is this a backward compatibility concern? I think it's fine, but people could have been getting 3 returned for things that had 4 dimensions if the measured part was being ignored. That was likely incorrect anyway, and I'm not sure how likely it is that this would have been encountered -  not sure if it's worth a note somewhere.

Reply:
I've added some words to the backwards incompatible changes section of  `5.1.txt` for this.

Reply:
I originally added a small release note... but on review now I'm not so sure. I don't think geometries with four dimensions could be created before this patch? 

```python
>>> OGRGeometry("POINT (1 2 3 4)")
TypeError: Unsupported geometry type: PointZM
```

Reply:
Agreed, it's not worth adding a note.